cmd/notation/verify.go

```diff
@@ -100,6 +110,10 @@ func runVerify(command *cobra.Command, opts *verifyOpts) error {
 		return err
 	}
 
+	if opts.outputFormat != ioutil.OutputJson && opts.outputFormat != ioutil.OutputPlaintext {
+		return fmt.Errorf("unrecognized output format: %v", opts.outputFormat)
+	}
+
```

Comment:
Should we do this as first thing in Verify operation like https://github.com/notaryproject/notation/blob/main/cmd/notation/sign.go#L76-L78?

Reply:
sure, makes sense to fail fast to avoid wasting effort on resolving the reference etc